opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// LibraryInfo is a message representing the instrumented library informations such as the fully
+// qualified name and version. 
+message LibraryInfo {
```

Comment:
Spec https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md#library ? also defines a language. Should it be defined here as well?

Reply:
Does the discussion from today sig meeting + https://github.com/open-telemetry/oteps/pull/84 + https://github.com/open-telemetry/opentelemetry-specification/pull/494 clarifies.

Reply:
@SergeyKanzhelev As discussed today, the linked resource convention is unrelated, so "language" does not apply here (yet).